Error `Cannot find module 'tslib'`

Running the following command:

```
npx ember-template-lint@beta --config='{ "extends": ["recommended", "a11y"] }' --no-inline-config
```

Results in an error:

```
Cannot find module 'tslib'
Require stack:
- /Users/rjackson/.npm/_npx/62256/lib/node_modules/ember-template-lint/node_modules/@ember-template-lint/todo-utils/lib/index.js
- /Users/rjackson/.npm/_npx/62256/lib/node_modules/ember-template-lint/bin/ember-template-lint.js
```

--- a/package.json
+++ b/package.json
@@ -18,7 +18,8 @@
   "dependencies": {
     "@types/eslint": "^7.2.6",
     "fs-extra": "^9.0.1",
-    "slash": "^3.0.0"
+    "slash": "^3.0.0",
+    "tslib": "^2.1.0"
   },
   "devDependencies": {
     "@types/fs-extra": "^9.0.6",

